x-pack/plugins/observability/server/lib/rules/custom_threshold/lib/create_timerange.test.ts

```diff
@@ -11,123 +11,73 @@ import moment from 'moment';
 import { createTimerange } from './create_timerange';
 
 describe('createTimerange(interval, aggType, timeframe)', () => {
-  describe('without timeframe', () => {
```

Comment:
Are these test changes only about removing tests without a timeframe? I am trying to understand if something else has changed here.

Reply:
I update the createTimerange function to always have a timeframe, so I just removed `describe('without timeframe', () => {}` that was wrapping the tests